### PR TITLE
Low-income use case

### PR DESCRIPTION
**Pull request by indexing**

### CODE REVIEW

**Comment by indexing** [comment]
I still need to add the sequence diagram for this use case, but I'll do that as a change after this file is merged. 

Additionally, I'd like this to display alphabetically by name in the list of pathways. Is this a change I can make post-merge? Not sure how to do it before.

**Comment by thekaveman** [comment]
@indexing the file is named `Low-income.md` which is why it is sorting above the all lowercase e.g. `agency-card.md`.

If you rename the file to be all lowercase like the rest, it will naturally sort.

File: docs/enrollment-pathways/Low-income.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+- The transit rider pays for transit rides with their debit or credit card, just as they do for groceries, a cup of coffee, or any other good or service.
+
+- The transit rider can enroll in a transit benefit from home when convenient; they do not have to visit a transit agency in person.
+
+- Secure state and federal solutions manage the transit rider’s personal identifiable information (PII): [Login.gov](Login.gov) and the California Department of Technology Identity Gateway (IdG). Transit riders do not have to share personal information with local transit operators.
+
+- Benefit enrollment takes minutes rather than days or weeks.
+
+- Benefit enrollment doesn’t require online accounts with private companies.
```
Comment (machikoyasuda) [inline docs/enrollment-pathways/Low-income.md:69]: These benefits (for Benefits, hah) are the same for all eligibility types. But I guess it's okay to repeat them.

One benefit that is SPECIFIC to low-income is that by using CalFresh status, the rider does not have to go through another income verification process, and the transit agency does not have to create an income verification process that they have to then manage and get approved by their Board of Directors and so on. And what if every transit agency had a different definition of low income? This allows for potentially having the _same_ definition of "low income" across the state.

**Comment by thekaveman** [comment]
I fully agree with all of @machikoyasuda's comments above!

File: docs/enrollment-pathways/Low-income.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+1. The transit rider visits the web application at [benefits.calitp.org](benefits.calitp.org) in a browser on their desktop computer.
+
+2. The transit rider chooses the transit operator that serves their area.
+
+3. The transit rider chooses to verify their eligibility as a participant in the [CalFresh Program](https://www.cdss.ca.gov/food-nutrition/calfresh).
+
+4. The transit rider authenticates with their existing [Login.gov](Login.gov) account or, if they don’t have one, creates a [Login.gov](Login.gov) account.
+
+5. The Cal-ITP Benefits app interfaces with the [California Department of Transportation](https://dot.ca.gov/) Identity Gateway (IdG) to verify benefit eligibility. The IdG uses personal information shared by [Login.gov](Login.gov) to verify Calfresh participation status.
```
[the remark this answers is not in the document]
Comment (angela-tran) [reply docs/enrollment-pathways/Low-income.md:23]: The Benefits app interfaces with the California Department of Technology - https://cdt.ca.gov/. (Their shared abbreviation makes this a common mistake.)

**Review by angela-tran — COMMENTED** [review]
Thanks for all your work on this @indexing . I left one comment about a small typo.

File: docs/enrollment-pathways/low-income.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+```
+1. The transit rider visits the web application at `benefits.calitp.org` in a browser on their desktop computer.
+
+2. The transit rider chooses the transit operator that serves their area.
+
+3. The transit rider chooses to verify their eligibility as a participant in the [CalFresh Program](https://www.cdss.ca.gov/food-nutrition/calfresh).
+
+4. The transit rider authenticates with their existing [Login.gov](Login.gov) account or, if they don’t have one, creates a [Login.gov](Login.gov) account.
+
+5. The Cal-ITP Benefits app interfaces with the [California Department of Transportation](https://dot.ca.gov/) Identity Gateway (IdG) to verify benefit eligibility. The IdG uses personal information shared by [Login.gov](Login.gov) to verify CalFresh participation status.
```
Comment (machikoyasuda) [inline docs/enrollment-pathways/low-income.md:57]:
```suggestion
5. The Cal-ITP Benefits app interfaces with the [California Department of Technology](https://cdt.ca.gov/) [Identity Gateway (IdG)](https://digitalidstrategy.cdt.ca.gov/primary-elements.html) to verify benefit eligibility. The IdG uses personal information shared by [Login.gov](Login.gov) to verify Calfresh participation status.
```

**Review by machikoyasuda — CHANGES_REQUESTED** [review]
Everything looks good except for that last CDT/Caltrans link typo.

**Review by machikoyasuda — APPROVED** [review]

**Pull request merged**